test/testdefs.jl

```diff
@@ -37,9 +37,11 @@ function runtests(name, path, isolate=true; seed=nothing)
                                  res_and_time_data[4],
                                  res_and_time_data[5])
         end
-        vcat(collect(res_and_time_data), rss)
-    finally
+        vcat(@show collect(res_and_time_data), rss)
```

Comment:
```suggestion
        vcat(@show(collect(res_and_time_data)), rss)
```

Reply:
Theoretically performance, since we almost never need the strings

Reply:
Oh, and because there's a number of consumers of `@test_throws` that inspect the result fields, so they're somewhat part of the API